docs/docs-components/Toc.js

```diff
@@ -122,7 +129,11 @@ export default function Toc({ cards }: Props): Node {
 
   const handleClick = (
```

Comment:
can we use named arguments? `({ hash,  event })`

Reply:
yes